other/bootstrap_daemon/tox-bootstrapd-tmpfile.conf

```diff
@@ -0,0 +1 @@
+d /var/run/tox-bootstrapd 0750 tox-bootstrapd tox-bootstrapd -
```

Comment:
Actually, ignore the comment that I just deleted here. Keep it as it is for now.